src/csvcubedpmd/models/rdf/pmdcat.py

```diff
@@ -4,16 +4,17 @@
 """
 import rdflib
 from rdflib import Namespace, URIRef, Literal
-from typing import Annotated
+from typing import Annotated, Optional
 from abc import ABC
 from datetime import datetime
 from csvcubedmodels.rdf.triple import Triple, InverseTriple, PropertyStatus
+from csvcubedmodels.rdf.datatypes import MARKDOWN
```

Comment:
Is MARKDOWN treated as a namespace, is that why it's in all caps?

Reply:
I stuck that there.